src/components/BarChart.js

```diff
@@ -89,6 +95,8 @@ const Rect = React.createClass({
         onClick={this.props.onClick.bind(null, this.props.value, this.props.label)}
         onMouseOut={this._handleMouseOut}
         onMouseOver={this._handleMouseOver.bind(null, this.props.label, this.props.value, this.props.x, y)}
+        rx={3}
+        ry={3}
```

Comment:
What do the `3` values do here? Do they correspond to `const shift = this.props.value > 0 ? '3px' : '-3px'` above? If so it may be good to give it a name.